### DISCUSSION TITLE
Is there still a 2.2 or 3.0 release/tag coming?

### DISCUSSION THREAD

**Discussion by benfishbus** [fivefilters/readability.php#3] [Q&A] [answered]
Just wondering, it's been pretty quiet in here.

**Comment by fivefilters** [comment, marked as answer]
@benfishbus we've finally got 3.0.0 (which is basically what the master branch has been the last few months) and 3.1.0 which is an update with the minimum PHP version upped to 7.4 and also compatibility with PHP 8.1.
